plugins/out_stackdriver/stackdriver_conf.c

```diff
@@ -295,6 +324,15 @@ int flb_stackdriver_conf_destroy(struct flb_stackdriver *ctx)
         return -1;
     }
 
+    if (ctx->k8s_resource_type){
+        flb_sds_destroy(ctx->namespace_name);
```

Comment:
ctx->namespace_name would be nil for k8s_node type?

Is destory safe here?

Reply:
Yes it will be safe. As the deletion function will check that if the input is nil or not.